nav2_costmap_2d/test/integration/costmap_tester.cpp

```diff
@@ -103,9 +110,9 @@ void CostmapTester::compareCells(nav2_costmap_2d::Costmap2D& costmap, unsigned i
     double furthest_valid_distance = 0; // ################costmap.cell_inscribed_radius_ + cell_distance + 1;
     unsigned char expected_lowest_cost = 0; // ################costmap.computeCost(furthest_valid_distance);
     if(neighbor_cost < expected_lowest_cost){
-      ROS_ERROR("Cell cost (%d, %d): %d, neighbor cost (%d, %d): %d, expected lowest cost: %d, cell distance: %.2f, furthest valid distance: %.2f",
+      RCLCPP_ERROR(rclcpp::get_logger("costmap_tester"), "Cell cost (%d, %d): %d, neighbor cost (%d, %d): %d, expected lowest cost: %d, cell distance: %.2f, furthest valid distance: %.2f",
```

Comment:
Need to run the linting tools. Lines too long.

Reply:
I'd like to run linting tools on all the nav2_costmap_2d code in one PR. I made issue #396 to track